src/client/wrappers/styleWrappers/CardWrapper.js

```diff
@@ -1,36 +1,20 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Card, Text, theme } from '@aragon/ui'
+import { Card, Text } from '@aragon/ui'
 import styled from 'styled-components'
+import { Link } from '../../components/styled-components'
 
 const CardWrapper = ({ children, title, addMore, addSeparators, viewMode }) => (
-  <div
-    css={`
-      width: 100%;
-    `}
-  >
+  <div css="width: 100%">
     {title && (
-      <Text
-        css={`
-          padding: 7px 0;
-        `}
-        size="xlarge"
-      >
+      <Text css="padding: 7px 0" size="xlarge">
         {title}
       </Text>
     )}
```

Comment:
This seems short enough now to put on one line:

    {title && <Text css="padding: 7px 0" size="xlarge">{title}</Text>}

Reply:
Seems so, but it gets highlighted by eslint if it's in one line.